### PR TITLE
Lack of Public-Key-Pins Header updates

### PR DESCRIPTION
**Pull request by RRudder**

### CODE REVIEW

File: submissions/description/server_security_misconfiguration/lack_of_security_headers/public_key_pins/template.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+## Business Impact
+
+Not having a `Public-Key-Pins` header can lead to reputational damage and indirect financial loss to the business due to an advanced attacker’s ability to access data through a MitM attack. The degree of impact is dependent on the sensitivity of data being stored and transmitted by the application, and the sophistication of the attacker’s abilities.
+
+## Steps to Reproduce
+
+1. Enable a HTTP interception proxy, such as Burp Suite or OWASP ZAP
+1. Navigate to the following endpoint using a browser: {{URL}}
+1. Capture the request using the HTTP interception proxy
```
Comment (evildaemond) [inline submissions/description/server_security_misconfiguration/lack_of_security_headers/public_key_pins/template.md:17]: Clarifying a technical note, the request does not show HTTP headers, it's the response. Updating this to suit will help both readers of the report and the customer. 

1. Capture the request using the HTTP interception proxy and review the response

File: submissions/description/server_security_misconfiguration/lack_of_security_headers/public_key_pins/template.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Lack of Public-Key-Pins Header
+
+## Overview of the Vulnerability
+
+A lack of the HTTP response header for `Public-Key-Pins` can lead to sensitive user data being retrieved by an advanced attacker through Machine-in-the-Middle (MitM) attacks. There are multiple HTTP response headers used in communication between the server and client which can be implemented to improve security against well documented vulnerabilities. For example, the `Public-Key-Pins` security header is used in legacy versions of browsers to prevent clients from loading reflected XSS attacks.
```
Comment (evildaemond) [inline submissions/description/server_security_misconfiguration/lack_of_security_headers/public_key_pins/template.md:5]: Change to Machine-in-the-Middle (MitM) to Person-In-The-Middle (PitM) would be worth here

File: submissions/description/server_security_misconfiguration/lack_of_security_headers/public_key_pins/template.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+An advanced attacker can leverage a missing `Public-Key-Pins` header to forge certificates and gain access to data through a MitM attack.
```
Comment (evildaemond) [inline submissions/description/server_security_misconfiguration/lack_of_security_headers/public_key_pins/template.md:7]: Change to Machine-in-the-Middle (MitM) to Person-In-The-Middle (PitM) would be worth here

**Review by evildaemond — CHANGES_REQUESTED** [review]
Few technical change then LGTM

File: submissions/description/server_security_misconfiguration/lack_of_security_headers/public_key_pins/template.md (hunk 3 above)
Comment (RRudder) [reply submissions/description/server_security_misconfiguration/lack_of_security_headers/public_key_pins/template.md]: Thanks mate, GTG

**Review by evildaemond — APPROVED** [review]
LGTM

**Pull request merged**